src/main/resources/templates/layout/header.mustache

```diff
@@ -8,7 +8,10 @@
     <link rel="stylesheet" href="css/sanitize.css" />
```

Comment:
We fetch CSS files individually? Is this the proposed pipeline ticket that should fix this?